compiler/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "deno_compiler"
+version = "0.49.0"
+license = "MIT"
+description = "Compiles TypeScript and bundles JavaScript with the ability to create v8 snapshots"
+repository = "https://github.com/denoland/deno"
+authors = ["the Deno authors"]
+edition = "2018"
+
+exclude = [
+  "typescript/tests/*",
+  "typescript/src/*",
+  "typescript/scripts/*",
+  "typescript/doc/*",
+  "typescript/lib/*/*.json",
+]
+
+[lib]
+path = "lib.rs"
+
+[dependencies]
+base64 = "0.12.3"
+bytecount = "0.6.0"
+deno_core = { path = "../core", version = "0.53.0" }
+futures = "0.3.5"
+indexmap = "1.5.1"
+jsonc-parser = "0.12.2"
+lazy_static = "1.4.0"
+regex = "1.3.9"
+ring = "0.16.15"
+serde_json = "1.0.57"
+serde = { version = "1.0.115", features = ["derive"] }
+sourcemap = "6.0.1"
+swc_common = { version = "=0.9.1", features = ["sourcemap"] }
+swc_ecma_transforms = { version = "=0.19.7", features = ["react"] }
```

Comment:
That's already part of `swc_ecmascript` given `"transforms"` feature

Reply:
It wasn't...  It was panic'ing.  I will try again.

Reply:
Yeah, validated it again, if I remove that, I get:

```
error[E0432]: unresolved import `swc_ecmascript::transforms::react`
  --> compiler/ast.rs:40:5
   |
40 | use swc_ecmascript::transforms::react;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `react` in `transforms`
```

So it appears to not be included in `swc_ecmascript` and you have to include the other crate to have it be available.

Reply:
CC @kdy1 is there a better way to this?

Reply:
Sorry, I forgot adding cargo features to swc_ecmascript. I'm too good at forgotting thngs :(
I'll publish a patch soon.